admin/services/restart.service.js

```diff
@@ -129,21 +138,22 @@ restartService.getSubmittedRestarts = async schoolId => {
       return { ...restart, status }
     }
   }), p => !!p)
-  pupils.map(p => {
-    const record = latestPupilRestarts.find(l => l.pupilId === p.id)
+  await Promise.all(pupils.map(async p => {
+    const record = latestPupilRestarts.find(l => l.pupil_id === p.id)
     if (record) {
+      const reason = await pupilRestartDataService.sqlFindRestartReasonDescById(record.pupilRestartReason_id)
       restarts.push({
         id: record.id,
         pupilId: p.id,
-        reason: record.reason,
+        reason,
```

Comment:
please just add the property name, its far more readable and consistent with the other properties

Reply:
@GuyHarwood updated